conftest.py

```diff
@@ -24,6 +32,7 @@ def login_admin(self):
         """Log-in the admin user."""
         self.login('admin@example.com')
 
+    # TODO password=PWD
     def login(self, email, password='password'):
```

Comment:
Do you want to change that directly here? Or is there something more to do?

Reply:
Because login_admin and the admin tests use the login function as well, I thought that would be a good task for the test refactoring.